src/GeminiLookup.php

```diff
@@ -136,7 +135,14 @@ public function lookup(EntityInterface $entity) {
         $urls['fedora'],
         ['allow_redirects' => FALSE, 'headers' => ['Authorization' => $token]]
       );
-      $links = Psr7\parse_header($head->getHeader("Link"));
+      // phpcs:disable
+      if (class_exists(\GuzzleHttp\Psr7\Header::class)) {
+        $links = \GuzzleHttp\Psr7\Header::parse($head->getHeader('Link'));
+      }
+      else {
+        $links = \GuzzleHttp\Psr7\parse_header($head->getHeader('Link'));
+      }
+      //phpcs:enable
```

Comment:
Could use a comment around the `phpcs:disable`/`:enable` business... and/or just skip the particular sniff here... or further refactor? 

... Could we go so far as to `require` `guzzlehttp/psr7:^1.7`, so we can just use the `Header::parse()` bit?